timesketch/lib/analyzers/interface.py

```diff
@@ -762,14 +762,14 @@ class BaseIndexAnalyzer(object):
         timeline_id: The ID of the timeline the analyzer runs on.
         tagged_events: Dict with all events to add tags and those tags.
         emoji_events: Dict with all events to add emojis and those emojis.
+        sketch: A Sketch instance.
+        timeline_id: The ID of the timeline the analyzer runs on.
```

Comment:
That seems to be duplicate, is already mentioned in the docstring

Reply:
done